c/public/lib/source/t2cstr/t2cstr.c

```diff
@@ -254,18 +255,20 @@ static int srcSeek(t2cCtx h, long offset) {
         }                                                           \
     while (0)
 
-#define CHKSUBRBYTE(h)                               \
-    do                                               \
-        if (next == end) {                           \
-            if (h->src.offset >= h->src.endOffset) { \
-                if (h->aux->flags & T2C_IS_CFF2)     \
-                    goto do_subr_return;             \
-                return t2cErrSrcStream;              \
-            }                                        \
-            next = refill(h, &end);                  \
-            if (next == NULL)                        \
-                return t2cErrSrcStream;              \
-        }                                            \
+/* Check for a byte that follows the first byte of an operator or operand.
+   If we're curently empty, but more bytes are available, refill from buffer.
+   If we're completely out of bytes, throw an error.                         */
+
+#define CHECK_SUBSEQUENT_BYTE(h)                                    \
```

Comment:
This is fine. We might want to combine the two macro definitions by passing a boolean flag to another shared macro like:
```
#define CHECK_BYTE(h, is_first)                                         \
	    do                                                              \
	        if (next == end) {                                          \
	            if (h->src.offset >= h->src.endOffset) {                \
                if (is_first && (h->aux->flags & T2C_IS_CFF2))                    \
                    goto do_endchar;                                \
...
```
And pass TRUE or FALSE from the two.

Reply:
I feel very strongly about having two separate macros with names that clearly state what they do, so I am going to leave this as is.

Reply:
If so that's ok.